rockcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: sdcore-gui
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+version: "0.1"
+summary: SD-Core GUI
+description: |
+  SD-Core Graphical User Interface OCI Image.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  sdcore-gui:
+    override: replace
+    command: node client/.next/standalone/server.js
+    startup: enabled
+
+parts:
+  install-node:
```

Comment:
Why not use a stage-snap in the frontend part instead of having a standalone part to install node? 

Any why installing the binary directly here and using the snap in the part below?

Reply:
Also note that the `npm` plugin comes with the following parameters:
- npm-include-node
- npm-node-version

Reply:
I've tried them both, I didn't manage to make them work.